clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ClioDocument.scala

```diff
@@ -14,7 +12,7 @@ trait ClioDocument {
     * Clio's source of truth, so disaster recovery can determine the order in which
     * updates should be re-applied.
     */
-  val upsertId: UUID
+  val upsertId: String
```

Comment:
Maybe now would be a good time to define a type alias for the type of this field? Or, even better, use a custom type and get ourselves some type safety while we're at it.

Reply:
Later. That's an internal fix we can do at any time. I want to unblock ClioP1.

And remember this?
https://broadinstitute.slack.com/archives/C2B5Q266P/p1506443608000468

(We should leave the UUID parsing in place as we're sure to need it soon.)